etcdmain/gateway.go

```diff
@@ -77,6 +78,23 @@ func newGatewayStartCommand() *cobra.Command {
 	return &cmd
 }
 
+func stripSchema(eps []string) []string {
+	var endpoints []string
+
+	for _, ep := range eps {
+
+		if strings.HasPrefix(ep, "https://") {
```

Comment:
Oh, sorry. I meant:
```go
if u, err := url.Parse(ep); err == nil {
    ep = u.Host
}
endpoints = append(endpoints, ep)
```
It'll strip the schema off a valid url and leave anything else (e.g., etcd-node:2379) alone.